core/templates/dev/head/base.html

```diff
@@ -280,6 +281,16 @@
               {{ warnings_and_loader() }}
             </div>
 
+            <noscript>
+              <div class="oppia-page-cards-container">
+                <div class="md-default-theme oppia-page-card oppia-long-text">
+                  <h2>We Need JavaScript in Your Browser <img src="/images/general/sad_face.png" /> </h2>
```

Comment:
@seanlip  - I apologize. I didn't realize that the material-design font was already included.  I removed the icon files and added <i class="material-icons">&#xE811;</i> to represent the sad face.

I just push the code up.  If you get a chance, please review it. :)